alerts/alerts.go

```diff
@@ -77,8 +84,36 @@ func newManager(options ...Option) (*manager, error) {
 				return nil, typeAssertionError
 			}
 			m.ttl = v
+			// make alertsbgcrawltime to half of ttl, so that we get two scans atleast.
+			AlertsBgCrawlTime = time.Duration(math.Min(float64(AlertsBgCrawlTime), float64((uint64)(time.Second) * (m.ttl/2))))
 		}
 	}
+	_, err := sched.Instance().Schedule(
```

Comment:
This schedule will run on all the nodes in a cluster. We should instead move this logic into porx, and ensure that this runs only as part of a cluster co-ordinator.

Reply:
You can introduce a new API/function called Purge in this package and invoke that as part of a schedule defined in porx